Features/Stories/FieldTypes/TextlineFieldType.feature

```diff
@@ -0,0 +1,247 @@
+Feature: Test the validations done on fields from PlatformUI - text line fieldtype
+    In order to validate the text line fieldtype
+    As an Editor user
+    I need to be able to create and update content with text line fieldtypes
+
+    Background:
+       Given I am logged in as an 'Administrator' in PlatformUI
+
+    ##
+    # Validate the existence of expected fields from a field type when creating a content
+    ##
+    @javascript @common
+    Scenario: A Content item of an Content Type that has a Text Line Field Definition must have a Text Line Field
+        Given a Content Type with an "text line" Field exists
+        When I create a content of this type
+        Then I should see a "text line" field
+
+    @javascript @common
+    Scenario: When editing Content item, the label of a Text Line Field must be the corresponding Field Definition name
```

Comment:
Could be merged with the one above (it's about rendering the field editing form). But if this feature is common to any fieldtype, it must go in the ContentEdit feature (ping @dpobel)